int64validator/at_least_sum_of.go

```diff
@@ -0,0 +1,96 @@
+package int64validator
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-framework/path"
+	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+
+	"github.com/hashicorp/terraform-plugin-framework-validators/helpers/validatordiag"
+)
+
+var _ tfsdk.AttributeValidator = atLeastSumOfValidator{}
+
+// atLeastSumOfValidator validates that an integer Attribute's value is at least the sum of one
+// or more integer Attributes.
+type atLeastSumOfValidator struct {
+	attributesToSumPaths []path.Path
```

Comment:
How about using `path.Paths`?

Reply:
Also, I think you should move to `path.Expressions` here: the idea being, expressions can be relative and are also "mergeable".

Whoever uses the validator, can pass in all form of expressions, and then in the validator you can `.Merge()` them, as well as call `(State|Config|Plan).Matches()` to get the corresponding, matching paths.

This allows to use those validators also with `*` path expressions etc.